prow/plank/controller_test.go

```diff
@@ -219,6 +219,20 @@ func TestTerminateDupes(t *testing.T) {
 			complete:        false,
 			shouldTerminate: true,
 		},
+		{
+			name:            "old j3",
```

Comment:
How are these two different from the two above? Is it the order?

Reply:
Yes the order is important. The new test ordering failed before this fix because the ProwJob returned by `ReplaceProwJob` was always assigned to the index `i` in `pjs` instead of the index of `toCancel`.

Reply:
Whoops! Nice fix.